docs/content/logging-and-monitoring/service-insight.md

```diff
@@ -52,7 +52,7 @@ These statistics are returned as JSON:
 Response codes:
 
 * HTTP 200 OK - Service is healthy
-* HTTP 404 - No upstreams/VS found for the requested hostname
-* HTTP 503 Service Unavailable - The service is down (All upstreams/VS are "Unhealthy")
+* HTTP 404 - No upstreams/VS/TS found for the requested hostname / name
```

Comment:
Flagging this as peripheral concern of what @vepatel mentioned. If they're interchangeable, the only change I would make it to remove the spaces ("hostname/name"), otherwise this might need to change too.

Reply:
Updated